content/en/tracing/profiler/enabling/php.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Enabling the PHP Profiler
+kind: Documentation
+code_lang: php
+type: multi-code-lang
+code_lang_weight: 50
+further_reading:
+    - link: 'getting_started/profiler'
+      tag: 'Documentation'
+      text: 'Getting Started with Profiler'
+    - link: 'tracing/profiler/search_profiles'
+      tag: 'Documentation'
+      text: 'Learn more about available profile types'
+    - link: 'tracing/profiler/profiler_troubleshooting'
+      tag: 'Documentation'
+      text: 'Fix problems you encounter while using the profiler'
+---
+
+{{< site-region region="us5" >}}
+<div class="alert alert-warning">
+  The Continuous Profiler is not available for the Datadog {{< region-param key="dd_site_name" >}} site.
+</div>
+{{< /site-region >}}
+
+<div class="alert alert-warning">
+The Datadog PHP Profiler is currently in public beta. Datadog recommends evaluating the profiler in a non-sensitive environment before deploying in production.
+</div>
+
+Starting in tracer version 0.69.0, both the tracer and profiler can be installed simultaneously using the script `datadog-setup.php`.
+
+## Requirements
+
+The Datadog Profiler requires at least PHP 7.1. PHP 8.1 is not yet supported. ZTS builds of PHP are not supported, nor are debug builds.
+
+A 64-bit, Linux Operating System (OS) is required.
+
+{{< tabs >}}
+{{% tab "GNU C Linux" %}}
+
+An Operating System with glibc 2.17 or newer is needed. The following OS versions or newer meet this requirement:
+  - CentOS 7
+  - Debian 8, which has reached End Of Life (EOL).
+  - Ubuntu 14.04, which is EOL.
+
+Datadog recommends running a supported OS version.
```

Comment:
What does this mean?

Reply:
Two of the minimum versions of OS's there are EOL. I'm not sure the best way to say it, but I was trying to indicate that Datadog recommends running a not-EOL OS version.

Reply:
I updated the wording:

> Datadog recommends running an OS version which is not EOL.

What do you think?

Reply:
I'm like -0.1 on including a warning like this but not a big deal.